flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosTaskManagerParameters.java

```diff
@@ -135,6 +135,13 @@
                             "A comma separated list of URIs of custom artifacts to be downloaded into the sandbox"
                                     + " of Mesos workers.");
 
+    public static final ConfigOption<String> MESOS_TM_USER =
+            key("mesos.resourcemenager.tasks.user")
+                    .stringType()
+                    .defaultValue(System.getProperty("user.name"))
```

Comment:
```suggestion
.noDefaultValue()
```
That's a minor thing, but: What about removing the default value. This would be closer to the implementation that is used right now.

Reply:
The documentation needs to be updated accordingly saying that no user is explicitly set if not specified.

Reply:
Done